kubernetes/utils/create_from_yaml.py

```diff
@@ -60,46 +66,69 @@ def create_from_yaml(
         processing of the request.
         Valid values are: - All: all dry run stages will be processed
     """
+    if path.exists(yaml_file):
+        with open(path.abspath(yaml_file)) as f:
```

Comment:
this block of code is still unnecessary, see https://github.com/kubernetes-client/python/pull/795/files#r295811946

Reply:
Correct. I removed this.